### PR TITLE
Feature/jc 599 make a book catalog page with a new design

### PR DESCRIPTION
**Pull request by trinhnguyen-saritasa**

### CODE REVIEW

File: apps/vanilla/src/pages/index.css [hunk 1]
```diff
@@ -1,5 +1,386 @@
...
+	padding: var(--spacing-sm) var(--spacing-sm);
+	appearance: none;
+	background-image: url("https://static.thenounproject.com/png/1666929-200.png");
+	background-repeat: no-repeat;
+	background-position: right 0.7rem top 50%;
+	background-size: 16px 16px;
+}
+
+.filters__search-input {
+	position: absolute;
```
Comment (irina-baleevskih-saritasa) [inline apps/vanilla/src/pages/index.css:133]: Let's use padding, you already set absolute position to button

File: apps/vanilla/src/pages/index.css [hunk 2]
```diff
@@ -1,5 +1,386 @@
...
+	top: 10px;
+	left: 38px;
+	font-size: 14px;
```
Comment (irina-baleevskih-saritasa) [inline apps/vanilla/src/pages/index.css:136]: You use vars for spacing, colors and also you set value. May be we can leave common approach ?

File: apps/vanilla/src/pages/index.css [hunk 3]
```diff
@@ -1,5 +1,386 @@
...
+	position: absolute;
+	top: 10px;
+	left: 15px;
+	height: 20px;
+	width: 20px;
+	padding: 0;
+	margin: 0;
+	border: none;
+	background: none;
+	outline: none !important;
```
Comment (irina-baleevskih-saritasa) [inline apps/vanilla/src/pages/index.css:160]: We shouldn't hide the outline, also we should avoid !important

File: apps/vanilla/src/pages/index.css [hunk 4]
```diff
@@ -1,5 +1,386 @@
...
+	font-weight: bold;
+}
+
+.filters__checkbox-wrapper {
+	--gray: #636e72;
+	--very-light-gray: #eee;
+	--light-gray: #9098a9;
+	--x-light-gray: #dfe6e9;
+	/* stylelint-disable-next-line declaration-colon-newline-after */
+	--gradient: linear-gradient(
```
Comment (irina-baleevskih-saritasa) [inline apps/vanilla/src/pages/index.css:252]: It's small component and Ithink this style is redundant

File: apps/vanilla/src/pages/index.css [hunk 5]
```diff
@@ -1,5 +1,386 @@
...
+		#6236ff 83%,
+		#b620e0 100%
+	);
+}
+
+.filters__checkbox-wrapper label {
+	font-size: 1.35em;
+}
+
+.filters__checkbox-wrapper input {
```
Comment (irina-baleevskih-saritasa) [inline apps/vanilla/src/pages/index.css:268]: Let's use classes instead tags as selectors

File: apps/vanilla/src/pages/index.css [hunk 6]
```diff
@@ -1,5 +1,386 @@
...
+		content: "";
+		width: 0.65em;
+		height: 0.65em;
+		border-radius: 50%;
+		transform: scale(0);
+		transition: 120ms transform ease-in-out;
+		box-shadow: inset 1em 1em var(--form-control-color);
+	}
+
+	input[type="radio"]:checked::before {
```
Comment (irina-baleevskih-saritasa) [inline apps/vanilla/src/pages/index.css:358]: I didn't find radio input, may be we can remove it?

File: apps/vanilla/src/pages/index.html [hunk 7]
```diff
@@ -4,13 +4,146 @@
...
+            </article>
+            <article class="article">
+              <img alt="book 3" src="images/blue.jpg" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+          </div>
+        </div>
+        <div>
+          <h5 class="section__year">1980 YEAR</h5>
+          <div class="section__group">
```
Comment (irina-baleevskih-saritasa) [inline apps/vanilla/src/pages/index.html:35]: Let's use ul to grouped data

File: apps/vanilla/src/pages/index.css [hunk 8]
```diff
@@ -1,5 +1,386 @@
 @import url("../global/index.css");
 
 .title {
-  color: var(--heading-color-1);
+	color: var(--heading-color-1);
+}
+
+.header {
+	justify-content: center;
+	top: 0;
```
[the remark this answers is not in the document]
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/index.css:9]: I removed top: 0; it doesn't work any more. But I still use justify to keep the "Books app" center

File: apps/vanilla/src/pages/index.css [hunk 9]
```diff
@@ -1,5 +1,386 @@
...
+	height: var(--height-main-header);
+	display: flex;
+	flex-direction: column;
+}
+
+.content {
+	margin: 0;
+	padding: 0 var(--spacing-2xl);
+	height: 100vh;
```
[the remark this answers is not in the document]
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/index.css:18]: yeah, margin and height does not work any more. I removed them. But I keep padding to make my work looks like the design in figma.

File: apps/vanilla/src/pages/index.css (hunk 8 above)
[the remark this answers is not in the document]
Comment (irina-baleevskih-saritasa) [reply apps/vanilla/src/pages/index.css:9]: Yes, it's correct, I mean only 'top' style

File: apps/vanilla/src/pages/index.html [hunk 10]
```diff
@@ -4,13 +4,146 @@
...
+              <input type="checkbox" name="checkbox" />
+            </div>
+
+            <span class="filters__checkbox-text">Show comics</span>
+          </label>
+        </div>
+      </form>
+    </main>
+
+    <hr class="sparate" />
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.html:133]: Typo. Let's come up with a better name like: 
```suggestion
    <hr class="divider" />
```

File: apps/vanilla/src/pages/index.css [hunk 11]
```diff
@@ -1,5 +1,386 @@
...
+	font-size: medium;
+	font-weight: bold;
+}
+
+.filters__search {
+	position: relative;
+	width: auto;
+	font-weight: light;
+	height: 40px;
+	border-radius: 12px;
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.css:103]: Same with variable for border, something like this should do:
`--border-radius: 12px`
`--border-radius-lg: 24px;`

File: apps/vanilla/src/pages/index.css [hunk 12]
```diff
@@ -1,5 +1,386 @@
...
+	width: 100%;
+	border-radius: 12px;
+	background: #f0f0f0;
+	transition: all 0.3s ease;
+	border: none;
+	/* stylelint-disable-next-line property-no-vendor-prefix */
+	-webkit-appearance: none;
+	/* stylelint-disable-next-line property-no-vendor-prefix */
+	-moz-appearance: none;
+	padding: var(--spacing-sm) var(--spacing-sm);
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.css:124]:
```suggestion
	padding: var(--spacing-sm);
```

File: apps/vanilla/src/pages/index.css [hunk 13]
```diff
@@ -1,5 +1,386 @@
...
+	display: visible;
+}
+
+.filters__checkbox {
+	display: flex;
+	align-items: center;
+	cursor: pointer;
+}
+
+.content__main__filter__checkbox__input {
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.css:180]: Is this class used anywhere?

File: apps/vanilla/src/pages/index.css [hunk 14]
```diff
@@ -1,5 +1,386 @@
...
+	border-radius: 4px;
+	margin-right: 8px;
+	cursor: pointer;
+}
+
+.filters__checkbox-text {
+	font-size: small;
+}
+
+.content__footer__container {
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.css:196]: Please review your class usage, I see this class doesn't get used anywhere. Should remove them.

File: apps/vanilla/src/pages/index.css [hunk 15]
```diff
@@ -1,5 +1,386 @@
...
+		#f7b500 33%,
+		#6dd400 50%,
+		#0091ff 67%,
+		#6236ff 83%,
+		#b620e0 100%
+	);
+}
+
+.filters__checkbox-wrapper label {
+	font-size: 1.35em;
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.css:265]: Why this element have different size?

File: apps/vanilla/src/pages/index.css [hunk 16]
```diff
@@ -1,5 +1,386 @@
...
+.filters__checkbox-wrapper input {
+	/* stylelint-disable-next-line property-no-vendor-prefix */
+	-webkit-appearance: none;
+	/* stylelint-disable-next-line property-no-vendor-prefix */
+	-moz-appearance: none;
+	appearance: none;
+	width: 25px;
+	height: 25px;
+	/* stylelint-disable-next-line property-no-unknown */
+	border-raudis: 4px;
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.css:277]: Typo

File: apps/vanilla/src/pages/images/c.jpg
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/images/c.jpg:1]: Do we need this?

File: apps/vanilla/src/pages/index.css [hunk 17]
```diff
@@ -1,5 +1,386 @@
...
+	font: inherit;
+	border: 0.12em solid var(--light-gray);
+	margin-bottom: -0.125em;
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.css:280]: For other styling different from `font-size` please consider using `px`

File: apps/vanilla/src/pages/images/bookIcon.jpg
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/images/bookIcon.jpg:1]: Same

File: apps/vanilla/src/pages/images/cleancode.jpg
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/images/cleancode.jpg:1]: Please, remove unused assets, even if they are cute :(

File: apps/vanilla/src/pages/index.html [hunk 18]
```diff
@@ -4,13 +4,146 @@
...
+    <title>Book App</title>
   </head>
-  <body class="default-layout">
-    <h1 class="title">Main page ⚡</h1>
-    <div class="links">
-      <a href="example/">Go to example page</a>
-    </div>
+  <body class="content">
+    <header class="header">
+      <h5>📚 Books app</h5>
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.html:11]: Why do we start with an `h5`? Review this https://arc.net/l/quote/mgytdsbh

File: apps/vanilla/src/pages/index.html [hunk 19]
```diff
@@ -4,13 +4,146 @@
...
+    </header>
+    <hr class="sparate" />
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.html:13]:
```suggestion
    <hr class="separate" />
```

File: apps/vanilla/src/pages/index.css [hunk 20]
```diff
@@ -1,5 +1,386 @@
...
+}
+
+.content::-webkit-scrollbar {
+	display: none;
+}
+
+.section__year {
+	margin-bottom: 10px;
+	font-size: var(--font-size-md);
+	font-weight: lighter;
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.css:28]: Font style doesn't match with what we have in design

File: apps/vanilla/src/pages/index.css [hunk 21]
```diff
@@ -1,5 +1,386 @@
...
+	cursor: pointer;
+
+	& svg {
+		width: 20px;
+		height: 20px;
+		fill: #5a6674;
+	}
+}
+
+.sparate--disable {
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.css:170]: Please, take a look at your divider. It doesn't match color with design

File: apps/vanilla/src/pages/index.html [hunk 22]
```diff
@@ -4,13 +4,146 @@
...
+    <main class="main main--reverse">
+      <section class="section">
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.html:15]: The class name does not have any meaning, this should identify what this section represent.

File: apps/vanilla/src/pages/index.html [hunk 23]
```diff
@@ -4,13 +4,146 @@
...
+        <div>
+          <h5 class="section__year">2020 YEAR</h5>
+          <div class="section__group">
+            <article class="article">
+              <img alt="book 1" src="images/rfui.jpg" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+            <article class="article">
+              <img alt="book 2" src="images/debug.jpg" class="article__img" />
+              <p class="article__title">Master</p>
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.html:25]: Don't forget to use the same titles as in design

File: apps/vanilla/src/pages/index.html [hunk 24]
```diff
@@ -4,13 +4,146 @@
...
+              <img alt="book 8" src="images/DDV.jpeg" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+            <article class="article">
+              <img alt="book 9" src="images/7HABIT.jpg" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+          </div>
+        </div>
+        <div>
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.html:64]: You have duplicating `1980` section

File: apps/vanilla/src/pages/index.html [hunk 25]
```diff
@@ -4,13 +4,146 @@
...
+              <img alt="book 2" src="images/debug.jpg" class="article__img" />
+              <p class="article__title">Master</p>
+            </article>
+            <article class="article">
+              <img alt="book 3" src="images/blue.jpg" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+          </div>
+        </div>
+        <div>
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.html:33]: It should be `section` to be semantically correct

**Review by timofei-maximov-saritasa — CHANGES_REQUESTED** [review]

File: apps/vanilla/src/pages/index.html [hunk 26]
```diff
@@ -4,13 +4,146 @@
...
+
+    <hr class="sparate" />
+    <footer class="footer">
+      <div class="footer__first">
+        <div class="footer__info">
+          <span class="footer__info-title">Books app</span>
+          <span>© 2023</span>
+        </div>
+        <div class="footer__link">
+          <a>Github</a>
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.html:141]: Remember to add `href` when using `a` tag, if it's not do anything, set it like this `href="#"`

File: apps/vanilla/src/pages/index.html [hunk 27]
```diff
@@ -4,13 +4,146 @@
...
+              <p class="article__title">Author</p>
+            </article>
+            <article class="article">
+              <img alt="book 14" src="images/7HABIT.jpg" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+          </div>
+        </div>
+      </section>
+      <hr class="sparate sparate--disable" />
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/pages/index.html:94]:
```suggestion
      <hr class="sparate sparate--hide" />
```

**Review by daoduong-saritasa — CHANGES_REQUESTED** [review]
Please review all your usage of color, try not to use plain value such as `white`, consider create a variable for it for better maintainability.

File: apps/vanilla/src/pages/index.css (hunk 4 above)
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/index.css:252]: thank you. I removed them.

File: apps/vanilla/src/pages/index.html (hunk 18 above)
[the remark this answers is not in the document]
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/index.html:11]: Yeah, I changed it. Thank you.

File: apps/vanilla/src/pages/index.html [hunk 28]
```diff
@@ -4,13 +4,146 @@
...
+              <p class="article__title">Author</p>
+            </article>
+            <article class="article">
+              <img alt="book 9" src="images/7HABIT.jpg" class="article__img" />
+              <p class="article__title">Author</p>
+            </article>
+          </div>
+        </div>
+        <div>
+          <h5 class="section__year">1980 YEAR</h5>
```
[the remark this answers is not in the document]
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/index.html:65]: I found it had sticky on mobile. Can you check it again? I set position: sticky;

File: apps/vanilla/src/global/variables.css [hunk 29]
```diff
@@ -2,4 +2,29 @@
   --heading-color-1: #ffd700;
   --heading-color-2: #e9967a;
   --heading-color-3: #b0e0e6;
+
+  /* height */
+  --height-main-header: 64px;
+  --height-main-footer: 128px;
+  --gap-medium: 10px;
```
[the remark this answers is not in the document]
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/global/variables.css:9]: I made mistake here, I should have used spacing instead. I removed  it.

File: apps/vanilla/src/pages/index.css (hunk 20 above)
[the remark this answers is not in the document]
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/index.css:28]: Thank you. I fixed it.

File: apps/vanilla/src/pages/index.html (hunk 18 above)
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/index.html:11]: Yeah, I changed to h1 for the first main content.

File: apps/vanilla/src/pages/index.html (hunk 22 above)
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/index.html:15]: I changed it to divider (with other reviewers suggested) and I have just changed it color. So I kept it.

File: apps/vanilla/src/pages/index.html (hunk 27 above)
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/index.html:94]: Thank you

File: apps/vanilla/src/pages/images/c.jpg
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/images/c.jpg:1]: No. I removed it. Thank you

File: apps/vanilla/src/global/variables.css [hunk 30]
```diff
@@ -1,30 +1,40 @@
...
+	--heading-color-2: #e9967a;
+	--heading-color-3: #b0e0e6;
 
-  /* height */
-  --height-main-header: 64px;
-  --height-main-footer: 128px;
-  --gap-medium: 10px;
+	/* color */
+	--primary-color: #fff;
+	--second-color: #000;
```
Comment (daoduong-saritasa) [inline apps/vanilla/src/global/variables.css:8]:
```suggestion
--primary-color: #ffffff;
--second-color: #000000;
```

**Review by daoduong-saritasa — APPROVED** [review]
Great job :+1:

File: apps/vanilla/src/pages/images/bookIcon.jpg
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/images/bookIcon.jpg:1]: Do you need this image?

File: apps/vanilla/src/pages/index.css [hunk 31]
```diff
@@ -1,5 +1,356 @@
...
+	font-weight: 300;
+	height: 40px;
+	border-radius: 12px;
+	background: #f0f0f0;
+	font-size: medium;
+	overflow: hidden;
+}
+
+.filters__select {
+	color: gray;
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.css:118]: Let's use css variables for colors, I believe we have design token for this

File: apps/vanilla/src/pages/index.css [hunk 32]
```diff
@@ -1,5 +1,356 @@
...
+	font-size: medium;
+	font-weight: 300;
+	position: relative;
+	height: 40px;
+	width: 100%;
+	border-radius: 12px;
+	background: #f0f0f0;
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.css:125]: Same here, please check styles in figma for select

image

File: apps/vanilla/src/pages/index.css [hunk 33]
```diff
@@ -1,5 +1,356 @@
...
+	transition: all 0.3s ease;
+	border: none;
+	/* stylelint-disable-next-line property-no-vendor-prefix */
+	-webkit-appearance: none;
+	/* stylelint-disable-next-line property-no-vendor-prefix */
+	-moz-appearance: none;
+	padding: var(--spacing-sm) var(--spacing-sm);
+	appearance: none;
+	background-image: url("https://static.thenounproject.com/png/1666929-200.png");
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.css:134]: This asset can be downloaded from figma

image

File: apps/vanilla/src/pages/index.css [hunk 34]
```diff
@@ -1,5 +1,356 @@
...
+	color: var(--divider-color);
+}
+
+.filters__search-input {
+	position: absolute;
+	top: 10px;
+	left: 38px;
+	font-size: 14px;
+	background: none;
+	color: #5a6674;
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.css:150]: Same with css variables

File: apps/vanilla/src/pages/index.css [hunk 35]
```diff
@@ -1,5 +1,356 @@
...
+.footer__info {
+	gap: var(--spacing-sm);
+}
+
+.footer__second {
+	margin-left: auto;
+	color: var(--second-color);
+	text-decoration: none;
+	text-decoration-color: none;
+}
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.css:242]: What do `first` and `second` mean from a naming perspective? It's not clear. Let's improve class naming

File: apps/vanilla/src/pages/index.css [hunk 36]
```diff
@@ -1,5 +1,356 @@
...
+	padding: var(--spacing-sm) var(--spacing-sm);
+	appearance: none;
+	background-image: url("https://static.thenounproject.com/png/1666929-200.png");
+	background-repeat: no-repeat;
+	background-position: right 0.7rem top 50%;
+	background-size: 16px 16px;
+}
+
+.divider {
+	color: var(--divider-color);
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.css:141]: Color is not applied on chrome.

You can try using `border-color` here. But check it on firefox if you are using it.

**Review by timofei-maximov-saritasa — COMMENTED** [review]
Please, review your css variables usage. You have a lot of colors used directly. Some of these colors don't match with what we have in figma.

File: apps/vanilla/src/pages/images/bookIcon.jpg
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/images/bookIcon.jpg:1]: I removed all non-usage images. Thank you sir.

File: apps/vanilla/src/pages/index.css (hunk 35 above)
Comment (trinhnguyen-saritasa) [reply apps/vanilla/src/pages/index.css:242]: Yeah, thank you sir. I changed it.

File: apps/vanilla/vite.config.ts [hunk 37]
```diff
@@ -21,8 +21,6 @@ export default defineConfig({
     rollupOptions: {
       input: [
         resolve(root, 'index.html'),
-        resolve(root, 'example', 'index.html'),
-        resolve(root, 'example', 'nested', 'index.html'),
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/vite.config.ts:25]: Don't forget to add `book-catalog` and `blackjack` pages here. 
This deleted code snippet was an example of adding multiple `html` files. Currently, preview is not working.

You can test it locally by running `npm run vanilla:build` and then running output locally.

File: apps/vanilla/src/pages/index.html [hunk 38]
```diff
@@ -1,16 +1,28 @@
...
-  </body>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="./index.css" />
+		<title>Main Page</title>
+	</head>
+	<body class="body">
+		<header >
+			<h1 >Main Page</h1>
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.html:11]:
```suggestion
			<h1>Main Page</h1>
```

File: apps/vanilla/src/pages/index.html [hunk 39]
```diff
@@ -1,16 +1,28 @@
...
-    <script type="module" src="./index.ts"></script>
-  </body>
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="./index.css" />
+		<title>Main Page</title>
+	</head>
+	<body class="body">
+		<header >
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.html:10]:
```suggestion
		<header>
```

File: apps/vanilla/src/pages/index.html [hunk 40]
```diff
@@ -1,16 +1,28 @@
...
+			<h1 >Main Page</h1>
+		</header>
+
+		<main >
+			<h1 >
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.html:15]:
```suggestion
		<main>
			<h1>
```

File: apps/vanilla/src/pages/index.html [hunk 41]
```diff
@@ -1,16 +1,28 @@
...
+				<a class="link-to-page" href="/book-catalog/index.html"> Go to Book Catalog Page </a>
+			</h1>
+
+			<h1 >
```
Comment (timofei-maximov-saritasa) [inline apps/vanilla/src/pages/index.html:19]:
```suggestion
			<h1>
```

**Review by timofei-maximov-saritasa — APPROVED** [review]

File: apps/vanilla/vite.config.ts (hunk 37 above)
Comment (trinhnguyen-saritasa) [reply apps/vanilla/vite.config.ts:25]: thank you

**Review by TienLuu-Saritasa — APPROVED** [review]

**Pull request merged**